tests/unit/checkboxradio/checkboxradio_methods.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * checkboxradio_methods.js
+ */
+
+
+(function($) {
+
+module( "Checkboxradio: methods" );
+	test( "Checkbox: refresh", function() {
+		var checkbox = $( "#checkbox-method-refresh" ),
+			widget;
+		expect( 2 );
+		checkbox.checkboxradio({
+			icon: true
+		});
+
+		widget = checkbox.checkboxradio( "widget" );
+		delete $(".ui-icon" )[0];
```

Comment:
Looks like this is supposed to remove the element from the DOM. That's not how it works.